code/game/objects/items/devices/radio/radio.dm

```diff
@@ -282,7 +282,7 @@
 	var/datum/signal/subspace/vocal/signal = new(src, freq, speaker, language, message, spans, message_mods)
 
 	// Independent radios, on the CentCom frequency, reach all independent radios
-	if (independent && (freq == FREQ_CENTCOM || freq == FREQ_CTF_RED || freq == FREQ_CTF_BLUE))
```

Comment:
bad space removals, put them back. unnecessary changes make for more merge conflicts

Reply:
I mean, consistency. Half the ifs have spaces between the bracket, half don't. It's annoying. I'll put it back if you insist.

Reply:
I do insist, you changed one line in the file but changed 20 others for an unenforced code standard